.github/to-russians.md

```diff
@@ -0,0 +1,15 @@
+## 🇷🇺 Русским гражданам
+
+Мы, участники Laminas, живем в разных странах. У многих из нас есть друзья, родственники и коллеги как в России, так и в Украине. Некоторые из нас родились в России. Некоторые из нас живут в России. У некоторых есть бабушки и дедушки, которые сражались с фашистами во Второй мировой войне. Здесь никто не поддерживает фашизм.
```

Comment:
I would like to adjust translation a bit to be more in line with Russian language.
```suggestion
Мы, команда Laminas, из разных стран всего мира. У многих из нас есть друзья, родственники и коллеги как в России, так и в Украине. Некоторые из нас родились в России. Некоторые из нас живут в России. У некоторых бабушки и дедушки сражались с фашистами во Второй мировой войне. Здесь никто не поддерживает фашизм.
```